packages/fuels/src/cli/utils/fuelsVersionCache.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs';
+import path from 'path';
+
+export const FUELS_VERSION_CACHE_FILE = path.join(__dirname, '.fuels-cache.json');
+
+export type FuelsVersionCache = {
+  data: { version: string } | null;
+  timestamp: number;
+};
```

Comment:
Do we need the `data`? I also think `version` should never be null.

```suggestion
export type FuelsVersionCache = {
  version: string;
  timestamp: number;
};
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3186/commits/7c4d96fe73014f0aa0abf07a18ef3d69aadbe68a